os/board/rtl8721csm/src/Makefile

```diff
@@ -142,13 +166,32 @@ CFLAGS += -I$(COMPONENT_DIR)/common/drivers/wlan/realtek/wlan_ram_map/rom
 CFLAGS += -I$(COMPONENT_DIR)/os/os_dep/include
 CFLAGS += -I$(COMPONENT_DIR)/os/tizenrt
 CFLAGS += -I$(TOPDIR)/net/lwip
-CFLAGS += -I$(TOPDIR)/net/lwip/src/include/
+CFLAGS += -I$(TOPDIR)/net/lwip/src/include
 CFLAGS += -I$(DIR_WIFI)/rtw_wpa_supplicant/src
 CFLAGS += -I$(DIR_WIFI)/rtw_wpa_supplicant/wpa_supplicant
 CFLAGS += -I$(COMPONENT_DIR)/soc/realtek/amebad/misc
-CFALGS += -I$(DIR_CRYPTO)/../include/
-CFLAGS += -I$(DIR_FW)/../include/
-CFLAGS += -I$(TOPDIR)/net/netmgr/
+CFALGS += -I$(DIR_CRYPTO)/../include
+CFLAGS += -I$(DIR_FW)/../include
+CFLAGS += -I$(TOPDIR)/net/netmgr
+ifeq ($(CONFIG_AMEBAD_BLE),y)
+CFLAGS += -I$(DIR_BT)/board/amebad/lib
+CFLAGS += -I$(DIR_BT)/board/amebad/src
+CFLAGS += -I$(DIR_BT)/board/amebad/src/vendor_cmd
+CFLAGS += -I$(DIR_BT)/board/common/inc
+CFLAGS += -I$(DIR_BT)/inc
+#CFLAGS += -I$(DIR_BT)/inc/app
+#CFLAGS += -I$(DIR_BT)/inc/bluetooth/gap
+#CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile
+#CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile/client
+#CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile/server
+#CFLAGS += -I$(DIR_BT)/inc/os
+#CFLAGS += -I$(DIR_BT)/inc/platform
+#CFLAGS += -I$(DIR_BT)/inc/stack
+CFLAGS += -I$(DIR_BT)/src/mcu/module/data_uart_cmd
+CFLAGS += -I$(DIR_BT)/example/da_ble_peripheral
+CFLAGS += -I$(DIR_BT)/example/da_ble_central
```

Comment:
Do you need to add example path?

Reply:
Yes.There are some head files in example path. I will fix error for the length limitation of CFLAGS by make other changes.